src/Psalm/Internal/Codebase/Populator.php

```diff
@@ -601,255 +691,152 @@ function ($constant) {
         $storage->pseudo_methods += $parent_storage->pseudo_methods;
     }
 
-    private function populateInterfaceDataFromParentInterfaces(
+    private function populateInterfaceData(
```

Comment:
This is common code pulled out of `populateInterfaceDataFromParentInterfaces` and `populateDataFromImplementedInterfaces`.